src/coreclr/tools/aot/Directory.Build.props

```diff
@@ -2,5 +2,6 @@
   <Import Project="../Directory.Build.props" />
   <PropertyGroup>
     <IsTrimmable>true</IsTrimmable>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
This will also exclude crossgen2 that you want to keep for source build. This would need to be done differently.